pkg/rke/k8s_rke_system_images.go

```diff
@@ -9848,6 +9848,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.8.1
```

Comment:
Enabled with Rancher v2.8.3, 1.28 won't be released out of band

Reply:
Updated